Jenkinsfile

```diff
@@ -7,7 +7,19 @@ node {
 
   stage('Build') {
     // you should build this repo with a maven build step here
-    echo "hello"
-  }
+        withMaven (maven: maven3) {
+            sh "mvn package"    }
   // you should add a test report here
-}
+  node {
```

Comment:
This extra `node` block will cause you problems when you can finally run your job.  It tells Jenkins to run this part of the job on a different node than the others, which means it will get a separate workspace.  That means the part that downloaded the code won't have executed on this node.  It's fixable by just not having this.

Reply:
Pushed some changes for jenkinsfile, hoping that fixed it